docs/example-react-hooks-useReducer.md

```diff
@@ -0,0 +1,119 @@
+Basic example showing how to test useReducer hook. 
+
+
+The component: changes stateprop1 from false to true which changes some text. 
+
+```
+import React, { useReducer } from 'react';
```

Comment:
As far as I can tell, most examples in Testing Library docs aren't using the trailing comma, so I'd suggest sticking to it :)

Reply:
Prettier _should_ format the docs on commit, I'm assuming the author wrote this on GH instead of locally?